offline/QA/Jet/photonjetskinematics.h

```diff
@@ -0,0 +1,72 @@
+// Tell emacs that this is a C++ source
+//  -*- C++ -*-.
+#ifndef PHOTONJETSKINEMATICS_H
+#define PHOTONJETSKINEMATICS_H
+
+#include <fun4all/SubsysReco.h>
+
+#include <string>
+#include <TFile.h>
+#include <TH1.h>
+#include <TH2D.h>
+
+class PHCompositeNode;
```

Comment:
We'll need to do a forward declaration for the hist manager...
```suggestion
class Fun4AllHistoManager;
class PHCompositeNode;
```